mosip-identity-plugin/src/main/java/io/mosip/esignet/plugin/mosipid/dto/IdaKycAuthResponse.java

```diff
@@ -5,12 +5,19 @@
  */
 package io.mosip.esignet.plugin.mosipid.dto;
 
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.databind.JsonNode;
 import lombok.Data;
 
+import java.util.List;
+import java.util.Map;
+
 @Data
+@JsonInclude(JsonInclude.Include.NON_NULL)
```

Comment:
Why is this annotation required here?

Reply:
Referred the TokenResponse DTO and there it was added for a field so added for response as well, but any how the verified claims will be requested based on claimsMetadata flag. Hence removed it from the response DTO.